docs/advanced-usage/persistent-storage.md

```diff
@@ -25,6 +25,12 @@ dokku storage:mount app-name /var/lib/dokku/data/storage:/storage
 
 Dokku will then mount the shared contents of `/var/lib/dokku/data/storage` to `/storage` inside the container.
 
+You will also need to restart the deployment, to activate storage:
+
+```shell
+dokku deploy app-name
```

Comment:
Sure! Changed in both places. I'm not sure how to refer to docs, so just wrote (see ps) with link.